core/src/banking_stage/consumer.rs

```diff
@@ -24,7 +24,10 @@ use {
         transaction_batch::TransactionBatch,
         verify_precompiles::verify_precompiles,
     },
-    solana_runtime_transaction::instructions_processor::process_compute_budget_instructions,
+    solana_runtime_transaction::{
```

Comment:
Is there a reason that `execute_and_commit_transactions_locked` is recomputing the compute budget details still rather than using them from runtime tx?